src/components/KeywordsInput.js

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from "react";
+
+const KeywordInput = () => {
+  const handleChange = e => {
+    setKeywords(e.target.value);
+  };
+
+  const [keywords, setKeywords] = useState("");
+
+  return (
+    <label>
+      <span className="label-text">Keywords</span>
+      <input value={keywords} onChange={handleChange} />
+    </label>
+  );
+};
+
+export default KeywordInput;
```

Comment:
This will definitely help simplify things by having each component in its own file.